.gitignore

```diff
@@ -11,7 +13,7 @@
 /composer.phar
 /coverage
 /node_modules/*
-/vendor/*
+/vendor/
```

Comment:
why?

Het formaat met een * achter heeft een betere ondersteuning in een aantal cli tools (zoals ack, ag,...)